cmd/proxy/actions/app_proxy.go

```diff
@@ -74,12 +74,23 @@ func addProxyRoutes(
 	// 3. The stashpool manages limiting concurrent requests and passes them to stash.
 	// 4. The plain stash.New just takes a request from upstream and saves it into storage.
 	fs := afero.NewOsFs()
-	mf, err := module.NewGoGetFetcher(c.GoBinary, c.GoProxy, fs)
+
+	// TODO: remove before we release v0.7.0
+	if c.GoProxy != "direct" && c.GoProxy != "" {
+		fmt.Println("GoProxy is deprecated, please use GoBinaryEnvVars")
```

Comment:
@marwan-at-work - use a logger here? We can do this in a follow-up PR if you like

Reply:
@arschles good point, i'll use it now